stream_alert/shared/logger.py

```diff
@@ -38,6 +38,6 @@ def get_logger(name, level=None):
         logger.setLevel(level.upper())
     except (TypeError, ValueError) as err:
         logger.setLevel('INFO')
-        logger.error('Defaulting to INFO logging: %s', err.message)
+        logger.error('Defaulting to INFO logging: %s', str(err))
```

Comment:
is the cast to `str` necessary here? won't the `%s` formatting directive handle this?